main.go

```diff
@@ -39,20 +39,37 @@ type ConfigsModel struct {
 }
 
 func createConfigsModelFromEnvs() ConfigsModel {
-	return ConfigsModel{
-		BuildArtifactPath:  os.Getenv("apk_path"),
+	cfg := ConfigsModel{
 		KeystoreURL:        os.Getenv("keystore_url"),
 		KeystorePassword:   os.Getenv("keystore_password"),
 		KeystoreAlias:      os.Getenv("keystore_alias"),
 		PrivateKeyPassword: os.Getenv("private_key_password"),
 		JarsignerOptions:   os.Getenv("jarsigner_options"),
 	}
+
+	if val := os.Getenv("apk_path"); val != "" {
+		log.Warnf("APK_PATH env detected. APK_PATH will be deprecated in future versions! Please use ANDROID_APP instead. Using APK_PATH value for current build.")
```

Comment:
pls use deprecation log: https://github.com/bitrise-steplib/steps-google-play-deploy/blob/master/config.go#L124
- with the same removal date and layout